src/ray/core_worker/core_worker.h

```diff
@@ -25,10 +26,9 @@
 /// 1) Add the rpc to the WorkerService in core_worker.proto, e.g., "ExampleCall"
 /// 2) Add a new handler to the macro below: "RAY_CORE_WORKER_RPC_HANDLER(ExampleCall, 1)"
 /// 3) Add a method to the CoreWorker class below: "CoreWorker::HandleExampleCall"
-#define RAY_CORE_WORKER_RPC_HANDLERS                       \
-  RAY_CORE_WORKER_RPC_HANDLER(AssignTask, 5)               \
-  RAY_CORE_WORKER_RPC_HANDLER(DirectActorAssignTask, 9999) \
-  RAY_CORE_WORKER_RPC_HANDLER(DirectActorCallArgWaitComplete, 100)
+#define RAY_CORE_WORKER_RPC_HANDLERS         \
+  RAY_CORE_WORKER_RPC_HANDLER(AssignTask, 5) \
+  RAY_CORE_WORKER_RPC_HANDLER(DirectActorAssignTask, 9999)
```

Comment:
I'm not sure removing the arg wait callback RPC is a good idea, unless we are fine with very large number of async RPCs to the node manager (like 100k+).